Sources/Templates/Extensions/Method+Extension.swift

```diff
@@ -284,11 +323,17 @@ private extension Method {
 
     func mockReturnType(type: Type) -> String? {
         guard !returnTypeName.isVoid else { return nil }
+
+        var mutableReturnTypeName = "-> \(returnTypeName.name)"
```

Comment:
Instead of this we could just return right?
```swift
    func mockReturnType(type: Type) -> String? {
        guard !returnTypeName.isVoid else { return nil }

        // We have to return a concrete type instead of `Self`
        if returnTypeName.name == "Self" {
            return "-> Default\(type.name)Mock"
        } else if returnTypeName.isOpaqueType && returnTypeName.isOptional {
            return "-> \(returnTypeName.wrapOptionalIfNeeded())"
        }
        return "-> \(returnTypeName.name)"
    }
```